openapi.yaml

```diff
@@ -19161,6 +19233,27 @@ components:
         Node Pools consist of a Linode type, the number of Linodes to
         deploy of that type, and additional status information.
       properties:
+        autoscaler:
+          type: object
+          description: |
+            When enabled, the number of nodes autoscales within the defined minimum and maximum values.
+          properties:
+            enabled:
+              type: boolean
+              description: Whether autoscaling is enabled for this Node Pool.
```

Comment:
Should these descriptions also include the defaults?

Reply:
@alsaihn It looks like the defaults are already listed in the description. Are you referencing something else?

Reply:
@alsaihn oh nevermind, this bit is missing the defaults. Will add them in!